sunpy/lightcurve/sources/stereo.py

```diff
@@ -0,0 +1,906 @@
+# -*- coding: utf-8 -*-
+"""STEREO LightCurve sources subclass definitions"""
+
+from __future__ import absolute_import
+
+__authors__ = ["Ankit Kumar"]
+__email__ = "ankitkmr.iitk@gmail.com"
+
+# This module was developed with funding from 
+# Google Summer of Code 2015
+
+
+from datetime import timedelta,datetime
+
+import pandas as pd
+from pandas import DataFrame
+import matplotlib.dates
+from matplotlib import pyplot as plt
+import numpy as np
+
+from astropy.io import ascii
+from astropy.table import Table, Column
+from astropy.utils import OrderedDict
+from astropy.table.column import MaskedColumn
+
+from sunpy.time import TimeRange, parse_time
+from sunpy.lightcurve import LightCurve
+
+
+__all__ = ['LETLightCurve', 'SITLightCurve', 'PLASTICLightCurve', 'SEPTLightCurve', 'HETLightCurve']
+
+
+#This code for _to_pandas is repeated here and in stereo.py for performance reasons.
+def _to_pandas(self):
```

Comment:
Is it the same function everywhere? would not be easier (and better) to keep it in a separate file and add use it in each of the ones needed?